### DISCUSSION TITLE
How does a setTopBox differ from an AppleTV?

### DISCUSSION THREAD

**Discussion by catongates** [maxileith/homebridge-appletv-enhanced#194] [Q&A] [answered]
Does setting `"setTopBox": true` only change the icon displayed in Home, or are there other implications?

**Comment by maxileith** [comment, marked as answer]
It's only the icon that is different.

**Comment by catongates** [comment]
Thanks. I was hoping it might make the AppleTV appear in third party apps like Eve and Controller for HomeKit, but digging into closed issues I see that's not the case. Bummer.
